sklearn/cluster/_kmeans.py

```diff
@@ -1370,6 +1370,9 @@ class KMeans(_BaseKMeans):
 
     For a comparison between K-Means and MiniBatchKMeans refer to example
     :ref:`sphx_glr_auto_examples_cluster_plot_mini_batch_kmeans.py`.
+
+    For a demonstration of how K-Means can be used for generating initial seeds
```

Comment:
The idea of this example is to show how the function `kmeans_plusplus` can be called independently of a classifier. So I wouldn't link this example here. Given that the documentation for the `kmeans_plusplus` function is rather short and the example at the bottom of the documentation page can be found quite easily (see [here](https://scikit-learn.org/stable/modules/generated/sklearn.cluster.kmeans_plusplus.html#sklearn.cluster.kmeans_plusplus)), I would just remove this link here and not put it in the documentation of the function.